harper-core/src/linting/lint_group.rs

```diff
@@ -58,6 +58,7 @@ use super::in_on_the_cards::InOnTheCards;
 use super::inflected_verb_after_to::InflectedVerbAfterTo;
 use super::its_contraction::ItsContraction;
 use super::its_possessive::ItsPossessive;
+use super::progressive_needs_be::ProgressiveNeedsBe;
```

Comment:
Shouldn't it be moved aside pronoun_*

Here I feel the rule is added at the wrong place, assuming they are sorted alphabeticaly